### PR TITLE
th-280: Bug: When trying to create a shift for the driver, the 404 error (in Postman) and a 500 error (in swagger) appear for an unauthorized user

### PR DESCRIPTION
**Pull request by TetianaDoroshko**
Fixed authorization checking.
Fixed path in swagger.

### CODE REVIEW

File: backend/src/packages/shifts/shift.controller.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ class ShiftController extends Controller {
     this.addRoute({
       path: ShiftsApiPath.ROOT,
       method: 'POST',
-      authStrategy: AuthStrategy.INJECT_USER,
+      authStrategy: [AuthStrategy.VERIFY_JWT, AuthStrategy.INJECT_USER],
       validation: {
```
Comment (albsbz) [inline backend/src/packages/shifts/shift.controller.ts:43]: It seems that difference between AuthStrategy.VERIFY_JWT and AuthStrategy.INJECT_USER that first throws error on unauthorized user and the second one not, so we need to use only one of them

Comment (TetianaDoroshko) [reply backend/src/packages/shifts/shift.controller.ts:43]: It turns out that INJECT_USER only attach user to request. And if there isn't token, it attaches user=undefined and allowed to get to the private endpoints.

Comment (albsbz) [reply backend/src/packages/shifts/shift.controller.ts:43]: Then we can use only AuthStrategy.VERIFY_JWT

Comment (TetianaDoroshko) [reply backend/src/packages/shifts/shift.controller.ts:43]: Yes, you are right. Thank you.

**Review by kalinkaaaa14 — APPROVED** [review]

**Pull request merged**